### PR TITLE
[plugin.audio.eco99music] 0.0.6

### PR DESCRIPTION
**Pull request by noamsto**
### Description

Made eco99music addon to be compatible with Kodi 19. Originally introduced [here](https://github.com/xbmc/repo-plugins/pull/1338). 

### Checklist:
- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.audio.eco99music/addon.py [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+                'genre': 'Podcast',
+                'plot': items[item]['description']
+            })
+        li.setArt({
+            'thumb':
+            items[item]['album_cover'],
+            'poster':
+            items[item]['album_cover'],
+            'fanart':
+            os.path.join(ADDON_FOLDER, 'resources/media/fanart.jpg')
```
Comment (enen92) [inline plugin.audio.eco99music/addon.py]:
```suggestion
            os.path.join(ADDON_FOLDER, 'resources', 'media', 'fanart.jpg')
```

File: plugin.audio.eco99music/addon.xml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            Welcome to eco99music, an innovative music application based on the consumer possibilities of the listener and browser, and adapts edited music playlists for the current and relevant status of each listener. The app offers the consumers
+a unique and personalizes service, based on their location, activity, time of the day and their mood.
+Enjoy!
+…
+FYI - our content is available only in Israel.
+        </description>
+        <disclaimer lang="en_GB">This is an official ECO 99FM Kodi add-on</disclaimer>
+        <language>he</language>
+        <platform>all</platform>
+        <license>GNU GENERAL PUBLIC LICENSE. Version 3, 29 June 2007</license>
```
Comment (enen92) [inline plugin.audio.eco99music/addon.xml]:
```suggestion
        <license>GPL-3.0-only</license>
```

**Review by enen92 — CHANGES_REQUESTED** [review]
See comments, minors. Good to go afterwards.
Thanks for the update

**Comment by enen92** [comment]
Please check also travis failures: https://travis-ci.com/github/xbmc/repo-plugins/builds/218778708

File: plugin.audio.eco99music/addon.py (hunk 1 above)
Comment (noamsto) [reply plugin.audio.eco99music/addon.py]: Done

File: plugin.audio.eco99music/addon.xml (hunk 2 above)
Comment (noamsto) [reply plugin.audio.eco99music/addon.xml]: Done

**Review by enen92 — APPROVED** [review]
Sorry for the delay

**Pull request merged**